packages/core/src/components/ColorPalette/ColorDisplay.tsx

```diff
@@ -54,8 +54,8 @@ function MirrorfulMenuButton({
     />
   )
 }
-
-export function ColorDisplay({
+const MotionRow = motion(VariantRow)
```

Comment:
i don't think this works actually. I no longer see the animation. Could we just add back in `<motion.div>` top level here?

Reply:
Thank you for pointing that out Alex, I have added back `<motion.div>`.